packages/light-react/src/Accounts/CreateAccount/AccountWritePhrase/AccountWritePhrase.js

```diff
@@ -4,6 +4,7 @@
 // SPDX-License-Identifier: MIT
 
 import React, { Component } from 'react';
+import { FormField } from 'light-ui';
```

Comment:
Probably want it `@parity/light-ui` at some point - especially when we start publishing at some point